src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -37,7 +37,7 @@
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.6830001-df202c6c" />
     <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.3.0-SNAPSHOT" />
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="1.0.1" />
-    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="0.1.52-alpha" />
+    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="0.1.63-preview" />
```

Comment:
any reason for using -preview instead of moving to -beta then to -rc?

Reply:
Since we have no clear boundaries as to what `alpha` or `beta` is, we use `preview` just like we do in PowerShell proper.

We can switch to `rc` when we get closer to the preview.